src/licensing-client.js

```diff
@@ -8,15 +8,21 @@ let client = undefined;
 
 async function getLicensingClient() {
     core.debug('Getting Licensing Client...');
+    let useLicenseClient = false;
     const editorPath = process.env.UNITY_EDITOR_PATH;
-    const version = process.env.UNITY_EDITOR_VERSION || editorPath.match(/(\d+\.\d+\.\d+[a-z]?\d?)/)[0];
-    core.debug(`Unity Editor Path: ${editorPath}`);
-    core.debug(`Unity Version: ${version}`);
-    await fs.access(editorPath, fs.constants.X_OK);
+    if (editorPath) {
+        core.debug(`Unity Editor Path: ${editorPath}`);
+        await fs.access(editorPath, fs.constants.X_OK);
+        const version = process.env.UNITY_EDITOR_VERSION || editorPath.match(/(\d+\.\d+\.\d+[a-z]?\d?)/)[0];
+        core.debug(`Unity Version: ${version}`);
+        const major = version.split('.')[0];
+        // if 2019.3 or older, use unity hub licensing client
```

Comment:
// explain why

Reply:
the license client doesn't exist in the editor installation prior to 2020